SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -29,6 +30,63 @@ DEFINE_LOG_CATEGORY(LogSpatialGDKEditor);
 
 #define LOCTEXT_NAMESPACE "FSpatialGDKEditor"
 
+namespace 
+{
+
+bool CheckAutomationToolsUpToDate()
+{
+#if PLATFORM_WINDOWS
+	FString RunUATScriptName = TEXT("RunUAT.bat");
+	FString CmdExe = TEXT("cmd.exe");
+#elif PLATFORM_LINUX
+	FString RunUATScriptName = TEXT("RunUAT.sh");
+	FString CmdExe = TEXT("/bin/bash");
+#else
+	FString RunUATScriptName = TEXT("RunUAT.command");
+	FString CmdExe = TEXT("/bin/sh");
+#endif
+
+	FString UatPath = FPaths::ConvertRelativePathToFull(FPaths::EngineDir() / TEXT("Build/BatchFiles") / RunUATScriptName);
+	
+	if (!FPaths::FileExists(UatPath))
+	{
+		FFormatNamedArguments Arguments;
+		Arguments.Add(TEXT("File"), FText::FromString(UatPath));
+		FMessageDialog::Open(EAppMsgType::Ok, FText::Format(LOCTEXT("RequiredFileNotFoundMessage", "A required file could not be found:\n{File}"), Arguments));
+
+		return false;
+	}
+
+#if PLATFORM_WINDOWS
+	FString FullCommandLine = FString::Printf(TEXT("/c \"\"%s\" -list\""), *UatPath);
+#else
+	FString FullCommandLine = FString::Printf(TEXT("\"%s\" %s"), *UatPath, *CommandLine);
```

Comment:
No command line defined before.
Should be `-list` I assume?

Reply:
Oh, right.